src/textarea/index.jsx

```diff
@@ -63,6 +63,20 @@ class Textarea extends TextInputComponent {
     super.setValue(value, complete);
   }
 
+  /**
+   * @override
```

Comment:
maybe document why we override it here?

Reply:
Any idea what to write here?

I'm overriding here to make it behave as intended by the method name is-dom-value-equal-to. If I don't overwrite, then it's a defect.